src/nvim/ui_client.c

```diff
@@ -287,57 +283,62 @@ void ui_client_event_raw_line(GridLineEvent *g)
                (const schar_T *)grid_line_buf_char, grid_line_buf_attr);
 }
 
-/// Restarts the embedded server without killing the UI.
+static Array restart_args = ARRAY_DICT_INIT;
+
 void ui_client_event_restart(Array args)
```

Comment:
the docstring is worth having, no?

Reply:
The `@param args` part may be misleading, as they are arguments of the UI event,